### DISCUSSION TITLE
BarChart with dashed lines and rounded bars?

### DISCUSSION THREAD

**Discussion by ChristoRibeiro** [tremorlabs/tremor-npm#875] [General]
Hi, 

Really good jobs for the library :)

I use the BarChart component, and I am looking to use dashed lines instead of solid ones. Is it possible to do that easily?

Also, BarChart uses rectangular bars, is it possible to have bars with radius?

**Comment by kamigerami** [comment]
We also want to know if its possible to change the radius of the bars (top rectangle) to a bit more rounded like below:
![barchart_rounded](https://github.com/tremorlabs/tremor/assets/9321510/738f810a-98ac-4c79-b68e-58354865c034)

It's possible using recharts so it should be possible if we get support for it added right?

**Comment by severinlandolt** [comment]
With our copy-and-paste charts, this will be as easy as adding `strokeDasharray="5 5"` to the cartesian grid. → https://tremor.so/docs/visualizations/area-chart
